src/Authoring/cswinmd/Program.cs

```diff
@@ -0,0 +1,212 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace Generator

```

Comment:
Just a question. Isn't possible for new code use 
```
namespace Generator;
```
to avoid nesting.

Reply:
only for C# 10 and later. This builds with .net 5 which defaults to 9 I think... I do manually set the lang version in the project to preview so this _probably_ works, but feels low priority/impact (rest of the code in the repo uses the nesting style).

Reply:
Isn't .NET 5 about to reach its end of life? I think dropping .NET support is definitely doable.

Reply:
the whole repo (and others) are built on .net 5; once there is a move to update, it will be done for all projects at the same time (separate from this PR :)) .